README.md

```diff
@@ -32,6 +32,15 @@ Make sure to replace `DdnsKeyUser`, `DdnsKeyPass`, and the hostnames with your a
 
 Note: There are additional Environment Variables which can be viewed by running `docker run noip-duc --help`.
 
+3b. To restart in the backgorund autmatically, use the docker compose script:
```

Comment:
Please add additional context here to introduce Docker Compose.

Such as: 
```
### Docker Compose 

Alternatively, you can use our Docker Compose file included in the repository to restart automatically... 
```

And please check your spelling 😀